types/migrations/20220714123910_edited_collections_2.sql

```diff
@@ -0,0 +1,6 @@
+-- Add migration script here
+create type collection_section as enum ('Intro', 'Body');
+
+alter table collection_chapter
+    add section collection_section;
```

Comment:
I realized that this is a nullable column, which we don't want. Let's make this `not null` and add `default 'Body'` so that existing rows get the value of `Body`.